scripts/envDFL.sh

```diff
@@ -255,8 +256,10 @@ set_environment() {
     export DYNAWO_LIBIIDM_EXTENSIONS=$DYNAWO_INSTALL_DIR/lib
     export DYNAWO_RESOURCES_DIR=$DYNAFLOW_LAUNCHER_INSTALL_DIR/share:$DYNAWO_INSTALL_DIR/share/xsd
 
-    ld_library_path_prepend $DYNAFLOW_LAUNCHER_INSTALL_DIR/lib64 # For local DFL libraries, used only at runtime in case we compile in shared
-    ld_library_path_prepend $DYNAFLOW_LAUNCHER_INSTALL_DIR/lib # For local DFL libraries, used only at runtime in case we compile in shared
+    if [ $MODE -ne 2 ]; then
+        ld_library_path_prepend $DYNAFLOW_LAUNCHER_INSTALL_DIR/lib64 # For local DFL libraries, used only at runtime in case we compile in shared
```

Comment:
Could you please explain the rational behind this fix?

Reply:
The fix applies a different mode if we run the tests or the coverage.

It fixed the problem about the user being able to execute the option `build-tests-coverage` using a Release build. Before, the coverage used the user build to run. Now, the coverage uses the coverage build in `/build/coverage` which is always built in Debug. To do this, @gautierbureau removed the `$DYNAFLOW_LAUNCHER_INSTALL_DIR/lib64` and `$DYNAFLOW_LAUNCHER_INSTALL_DIR/lib` which was previously added to the `LD_LIBRARY_PATH`. This way, the coverage will only link with the .so files of the coverage which are built in Debug type. Previously, it searched for the .so files of the user build in the `LD_LIBRARY_PATH` which could be built in Release type... which leads to the error.